### PR TITLE
Start a draft of an embedded tutorial template

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
A draft template to reuse when rolling out tutorials for using Slint on Embedded devices.

### CODE REVIEW

File: docs/_templates/embedded-tutorials.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Embedded Tutorial template
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+
+<!-- Copy and use as a basis -->
+
+Start with an introduction to the tutorial. Include the following:
+
+-   What will the reader learn by the end of it?
+-   What hardware, and architecture combination is this tutorial designed for?
```
Comment (tronical) [inline docs/_templates/embedded-tutorials.md:10]: Yep, makes sense to mention that early on.

File: docs/_templates/embedded-tutorials.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Limitations
+
+Does this platform and architecture combination have any limitations, issues, etc?
```
Comment (tronical) [inline docs/_templates/embedded-tutorials.md:14]: Maybe "Known Limitations"? As a reader, that gives me hope that perhaps one day they will be lifted with a "software update" :)

File: docs/_templates/embedded-tutorials.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Prerequisites
+
+-   What must a reader have available or installed to follow this tutorial?
+    -   Slint language SDKs (seems obvious, but always worth providing people with a path back to earlier concepts)
```
Comment (tronical) [inline docs/_templates/embedded-tutorials.md:19]: Oh yes, there's nothing too obvious here.

**Review by tronical — COMMENTED** [review]
Structure looks good to me. Maybe a section at the end where to continue, i.e. get in touch with us about licensing for the embedded devices.

**Comment by ChrisChinchilla** [comment]
@tronical Made some updates

**Review by tronical — APPROVED** [review]
Would you like me to merge this as-is?

**Comment by ChrisChinchilla** [comment]
@tronical Sure, let's do it, I don't think GitHub has anyway to define content templates and it's not beinf rendered, so should be fine.

**Pull request merged**